src/App.js

```diff
@@ -1,18 +1,20 @@
 import React, { Component } from 'react';
-
+import Modal from 'react-modal';
 import './css/style.css';
 import Header from './header';
 import MainContent from './mainContent';
 import Nav from './nav';
 import Footer from './footer';
+import SingleMovieView from './singleMovieView';
 import movieData from './data/test-data';
 
 class App extends Component {
   constructor() {
     super();
     this.state = {
       movies: [],
-      filterOption: 'bangersToStankers'
+      filterOption: 'bangersToStankers',
+      selectedMovie: false,
```

Comment:
Typically, the datatype of the state value should not change because it can be confusing to developers (here it changes from a `boolean` to an `object`). The exception that I see most often is one you're doing here, though. However, instead of starting out at `false`, using `null` is more typical.